LayoutTests/fast/block/positioning/fixed-position-transform-related-container-expected.txt

```diff
@@ -0,0 +1 @@
+PASS if no assert in Debug.
```

Comment:
nit; Why not just a reftest? And, what's the behavior for a release build?

Reply:
hmm sorry what do you mean by reftest?

Reply:
Modified to *expected.html, please help to check if this works, thanks!